client/src/components/Hamburger.jsx

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components/macro'
+
+export default function Hamburger({ active, onClick }) {
+  return (
+    <Bun onClick={onClick} className={active ? 'active' : ''}>
```

Comment:
Same with the `onClick`. Also, why do you need empty classNames in the span elements?

Reply:
I most certainly do not... 😆 thank you! Not only for this but for all the comments. They have been very helpful - as always ☺️

Reply:
I am so happy that I could help 🚀 🙂